tailwind.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     removeDeprecatedGapUtilities: true,
     purgeLayersByDefault: true,
   },
-  purge: ['./components/**/*.{js,ts,jsx,tsx}', './pages/**/*.{js,ts,jsx,tsx}'],
+  purge: false,
```

Comment:
Had to turn off purging for tailwind for now, as it is botching the reach-ui dialog styles for some reason.